### PR TITLE
make it easier to run on macs

### PR DESCRIPTION
**Pull request by soenkehahn**

### CODE REVIEW

**Comment by soenkehahn** [comment]
#54 should be merged first.

**Comment by soenkehahn** [comment]
Fixes #43.

File: check-protocols-in-docker.sh [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+script=$1
+if [[ "$script" = /* ]]; then
+  docker run --rm -it --cap-add=SYS_PTRACE -v $script:/root/$(basename $script) -v $script.protocols.yaml:/root/$(basename $script).protocols.yaml check-protocols $(basename $script)
+else
+  echo argument must be relative
```
[the remark this answers is not in the document]
Comment (soenkehahn) [reply check-protocols-in-docker.sh]: Yes, of course.

**Comment by soenkehahn** [comment]
Yeah, if you try to mount a file that doesn't exist with `docker run`, `docker` creates a directory with the same name. :( This should be fixed in a separate PR. (#61)

**Comment by matthandlersux** [comment]
@soenkehahn ah, got it, and now i realize this is because my file was named `...protocol.yaml` not `...protocols.yaml`

**Pull request merged**